restful_resource.gemspec

```diff
@@ -27,11 +27,11 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec_junit_formatter'
 
   spec.add_dependency 'activesupport', '>= 6', '< 8'
-  spec.add_dependency 'faraday', '~> 1.10'
+  spec.add_dependency 'faraday', '>= 1.10'
   spec.add_dependency 'faraday-cdn-metrics', '~> 0.2'
   spec.add_dependency 'faraday-encoding'
   spec.add_dependency 'faraday-http-cache', '~> 2.2'
   spec.add_dependency 'faraday_middleware', '~> 1.0'
```

Comment:
This gem has been deprecated and is no longer supported, many of the core functions have been moved to Faraday or other gems. We should remove this so we can update our version of Faraday.

https://github.com/lostisland/faraday_middleware